test/alarmHistory.test.ts

```diff
@@ -1,50 +1,50 @@
-import { secondsBetween, AlarmState } from '../src/alarmHistory'
-import { DescribeAlarmHistoryOutput, AlarmHistoryItem } from 'aws-sdk/clients/cloudwatch';
+import { secondsBetweenFromHistory, AlarmState } from '../src/alarmHistory'
+import { DescribeAlarmHistoryOutput } from 'aws-sdk/clients/cloudwatch';
 
 describe('alarmHistory', () => {
   describe('#secondsBetween()', () => {
```

Comment:
You've changed the method name, the description needs to be updated.

Reply:
done